gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckJUnitDependencies.java

```diff
@@ -164,7 +163,7 @@ private boolean hasDep(Set<ResolvedComponentResult> deps, Predicate<ModuleVersio
     }
 
     private boolean sourceSetMentionsJUnit4(SourceSet ss) {
-        return !ss.getAllSource()
+        return !ss.getAllJava()
```

Comment:
Might be worth adding a small comment here that `getAllJava` is also including groovy files.